src/stream/src/executor/sink.rs

```diff
@@ -203,6 +205,12 @@ impl<F: LogStoreFactory> SinkExecutor<F> {
                 match msg? {
                     Message::Watermark(w) => watermark = Some(w),
                     Message::Chunk(c) => {
+                        actor_context
+                            .streaming_metrics
+                            .sink_input_row_count
+                            .with_label_values(&[&sink_id_str, &actor_id_str, &fragment_id_str])
+                            .inc_by(c.capacity() as u64);
```

Comment:
Note that the metrics is actually the **input** rows of Sink (previously, it's output). This is because we don't have a well definition for the output of sinks yet.

Technically, the input and output rows might be different because of `force_append_only` etc., but either can be a good reference.